lib/std/crypto/benchmark.zig

```diff
@@ -10,6 +10,8 @@ const crypto = std.crypto;
 const KiB = 1024;
 const MiB = 1024 * KiB;
 
+const block_size = 8 * 8192;
```

Comment:
Will remove this commit for the benchmark changes and rebase off master as this was added in another PR.